platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java

```diff
@@ -536,7 +537,7 @@ public class SwirldsPlatform implements Platform {
             eventStreamManagerName = String.valueOf(selfId);
         }
 
-        final EventStreamManager eventStreamManager = new EventStreamManager(
+        final EventStreamManager defaultEventStreamManager = new DefaultEventStreamManager(
```

Comment:
Nit: for other components, we've been leaving the variable name as-is, and not adding `default`. `defaultEventStreamManager` -> `eventStreamManager`

(probably not worth changing, unless you're also making other changes)

Reply:
fixed